rag/llm/sequence2txt_model.py

```
@@ -108,7 +120,62 @@ def transcription(self, audio_path):
+        }
+
+        try:
+            response = requests.post(url, headers=headers, json=payload, timeout=60)
+            response.raise_for_status()
+            result = response.json()
+            text = result.get("text") or result.get("output", {}).get("text")
+            if not text:
+                raise ValueError("Missing transcription text in Fun-ASR-Flash response")
+            text = text.strip()
+            return text, num_tokens_from_string(text)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)